client/python/gradio_client/client.py

```diff
@@ -603,6 +605,135 @@ def _get_config(self) -> dict:
                 )
             return config
 
+    def deploy_discord(
+        self,
+        discord_bot_token: str | None = None,
+        api_names: list[str] | None = None,
```

Comment:
Might be more foolproof for developers to pass in a dictionary of {"api_name": "command_name"} rather than have to align two separate lists

Reply:
Another option would be that `api_names` could be a `list[str | tuple[str, str]]`. If a tuple is provided, than the command name is the second entry. Would give flexibility to override particular api names but not necessarily all of them